src/client/components/localisation/utils.ts

```diff
@@ -33,7 +33,7 @@ const coloredMaterial = (material: Material, color?: string) => {
 }
 
 const parseConfig = createTransformer((config: any) => {
-  const { geometry, materials } = new JSONLoader().parse(config)
+  const { geometry, materials } = new LegacyJSONLoader().parse(config, '/')
```

Comment:
The types require a `path` to be given, but we have no external resources to load anyway. So it is not actually used.